### PR TITLE
is-lower-case-email-validation 

### PR DESCRIPTION
**Pull request by petushka1**
- Set restrictions for the email input
- If the restriction is met user is being disabled to submit a form
- UI: a custom error message is displayed below the input

### CODE REVIEW

**Review by nicupop729 — COMMENTED** [review]
Hi @petushka1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request



You have submitted a project, that was done without collaboration. This is a pair programming activity, so I kindly ask you to work with your project partners following the driver-navigator approach and all of you need to commit and push to this branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -138,17 +139,11 @@ <h2>
                 <button class="btn contact-btn" type="submit" name="btn">Get in touch</button>
```
Comment (OLIPLICHE) [inline index.html:139]: - [ ] Please the error message should show up close to the "Get in touch" button.
-
good

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @petushka1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️

- [ ] Please add a title to your pull request because your work need to be documented  [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (petushka1) [reply index.html:139]: hello, thanks for your input. But there is a title for this PR "Contact Form Validation Review #14"

Comment (OLIPLICHE) [reply index.html:139]: > hello, thanks for your input. But there is a title for this PR "Contact Form Validation Review #14"
s
- Please there is a link connected with "in a professional way" That is why it is in blue color. Kindly click here to learn more ([in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md))

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @petushka1   😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**